lib/cudadrv/libcuda.jl

```diff
@@ -1145,23 +1163,23 @@ end
 
 const CUasyncNotificationType = CUasyncNotificationType_enum
 
-struct var"##Ctag#299"
```

Comment:
Did you run this in a fresh session, i.e., `julia --project res/wrap.jl`? If not, that should hopefully remove some of these changes.

Reply:
I did:
```shell
cd CUDA.jl/res/wrap
julia --project
]up
include("wrap.jl")
main()
```

I just updated the PR to only update `CUDA_SDK_jll`.